polling_stations/templates/address_select.html

```diff
@@ -1,40 +1,32 @@
 {% extends "base.html" %}
 
 {% load i18n %}
+{% load dc_forms %}
 
 {% block content %}
 
-<div class="row">
-  <div class="columns large-12">
-    <div class="card">
-      <h2 id="address_picker">
-        {% trans "Choose Your Address / Street" %}
-      </h2>
-
-      <form method=post class="form form-inline">
-        {% csrf_token %}
-        <div class="form-group">
-            {{ form.as_p }}
-            <button type="submit" class="button" id="submit-address">
-              {% trans "Find my Polling Station" %}
-            </button>
-        </div>
-      </form>
-
-    </div>
-
-  </div>
+<div class="card">
+  <h2 id="address_picker">
+    {% trans "Choose Your Address / Street" %}
+  </h2>
+
+  <form method=post class="form form-inline">
+    {% csrf_token %}
+    {{ form|dc_form }}
+    <button type="submit" class="button" id="submit-address">
+      {% trans "Find my Polling Station" %}
+    </button>
+  </form>
 </div>
 
+
 {% endblock content %}
 
 {% block in_page_javascript %}
 <script type="text/javascript">
-$(document).ready(function() {
```

Comment:
This seems to work fine everywhere I tested it, but I'm just a bit confused by this change and the commit message for it. Your commit comment is "Don't require jQuery for raw JS" but one of the 4 lines of JS wrapped by this is `$('.select_multirow').css('-webkit-appearance', 'menulist');` which is using jQuery.

Also I was under the (possibly misguided) impression that anywhere you're using `$('thing').stuff()` it needs to be wrapped in a `ready()`.

This comment is a request for clarification/gap in my knowledge rather than a request for change.

Reply:
1. Whoops, I missed that call. I've changed it. FYI this is the first part in a project to remove jQuery as a dependency from our projects, or more generally get used to not using it. It's slightly pointless on this project as the other JS we pull in is going to require it.

2. `ready()` just prevents the JS from running before the HTML has rendered / downloaded. For a lot of cases the need for this goes when you put the JS below (in the source order) the HTML it's expecting. I think this is a good call for this case as the user will want the menulist to work before all images etc have loaded. Really we want to convert the style of the menulist as early as we can, so waiting for the document to be 'ready' isn't helpful.

Reply:
> remove jQuery as a dependency from our projects

Lets not get bogged down in it here as there is enough to discuss but can we earmark this as something to talk about in a bit more detail - would be useful to hear your thoughts on it. Maybe Tuesday would be a good time for that?